ruleset.xml

```diff
@@ -1,23 +1,5 @@
 <?xml version="1.0"?>
 <ruleset name="Zend Framework Coding Standard">
     <description>Zend Framework Coding Standard</description>
-
-    <!-- display progress -->
-    <arg value="p"/>
-    <arg name="colors"/>
-
-    <!-- inherit rules from: -->
-    <rule ref="PSR2"/>
-    <rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
-    <rule ref="Generic.Formatting.SpaceAfterNot"/>
-    <rule ref="Squiz.WhiteSpace.OperatorSpacing">
-        <properties>
-            <property name="ignoreNewlines" value="true"/>
-        </properties>
-    </rule>
-    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
-        <properties>
-            <property name="ignoreBlankLines" value="false"/>
-        </properties>
-    </rule>
+    <rule ref="vendor/zendframework/zend-coding-standard/src/ZendCodingStandard/ruleset.xml"/>
```

Comment:
> Have you tested another repo using this yet?

See the updated issue (first post). Tested so far on: zend-expressive, zend-code, zend-expressive-tooling, zend-mvc, zend-http

> what happens when a user decides to use a directory other than vendor

This is custom crafted for zend framework components. If someone manages to change the vendor location he needs to change composer.json the component.

Thinking about this, someone might want to use the coding standard for his own project. I could easily add a namespace check so the license will only trigger for `zendframework/<end-<component>`.

> my main concern is repositories already using it.

I haven't tested that. But I think the first travis build or local trigger of phpcs will fail. In that case running phpcbf and a new commit with the changes will fix the build again.